[2024-09-11T01:01:12.093+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-11T01:01:13.847+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2024-09-11T01:00:01+00:00 [queued]>
[2024-09-11T01:01:13.958+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: welcome_dag.print_date scheduled__2024-09-11T01:00:01+00:00 [queued]>
[2024-09-11T01:01:13.973+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-11T01:01:14.057+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): print_date> on 2024-09-11 01:00:01+00:00
[2024-09-11T01:01:14.121+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'welcome_dag', 'print_date', 'scheduled__2024-09-11T01:00:01+00:00', '--job-id', '432', '--raw', '--subdir', 'DAGS_FOLDER/Awelcome_dags.py', '--cfg-path', '/tmp/tmpu80_u6qx']
[2024-09-11T01:01:14.145+0000] {standard_task_runner.py:105} INFO - Job 432: Subtask print_date
[2024-09-11T01:01:14.166+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8503) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-11T01:01:14.180+0000] {standard_task_runner.py:72} INFO - Started process 8544 to run task
[2024-09-11T01:01:17.897+0000] {task_command.py:467} INFO - Running <TaskInstance: welcome_dag.print_date scheduled__2024-09-11T01:00:01+00:00 [running]> on host 5cd9362923f6
[2024-09-11T01:01:21.284+0000] {job.py:229} INFO - Heartbeat recovered after 14.52 seconds
[2024-09-11T01:01:24.605+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='welcome_dag' AIRFLOW_CTX_TASK_ID='print_date' AIRFLOW_CTX_EXECUTION_DATE='2024-09-11T01:00:01+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-11T01:00:01+00:00'
[2024-09-11T01:01:24.609+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-11T01:01:24.710+0000] {logging_mixin.py:190} INFO - Today is 2024-09-11
[2024-09-11T01:01:24.728+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-11T01:01:26.447+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-11T01:01:26.450+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=welcome_dag, task_id=print_date, run_id=scheduled__2024-09-11T01:00:01+00:00, execution_date=20240911T010001, start_date=20240911T010113, end_date=20240911T010126
[2024-09-11T01:01:26.632+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-11T01:01:28.074+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-11T01:01:28.097+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-09-11T01:01:29.246+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
